### ISSUE TITLE
VS Code gets very confused trying to import a standard library.

### ISSUE THREAD

**Issue by TradeIdeasPhilip** [denoland/vscode_deno#518]
**Describe the bug**

I tried adding `import { writeAll } from "https://deno.land/std@0.106.0/io/util.ts";` to a TypeScript file in a Deno project, and VS Code gave me all sorts of trouble.  Small changes to my file caused a number of different symptoms including:

- Sometimes I saw an error message when I shouldn't have.  (Multiple versions of this.)
- Sometimes the tooltips were inconsistent.
- Sometimes I could get rid of the errors, but `writeAll` was imported as `any`, not the correct type.

I tried all the usual things to fix it.  Like closing and reopening the tab, closing and reopening the entire window, closing all VS Code windows and restarting, reordering the code, and importing the code without the specific version number.  Some details changed but I could never get the import to work.

*See [the video](https://www.youtube.com/watch?v=Q2vG8PAK7Ek) for more details.*

**To Reproduce**

No idea.  I've imported lots of files in the past, including local files and standard library files.  I've never seen anything like this before.

**Expected behavior**

This was a standard library file.  It should have reported no errors.  TypeScript should have known the type of the item I was importing.  Adding one import should not cause others to fail.  The tooltips should report on the file under the cursor, not the file on the next line.

**Screenshots**

This picture shows two problems.  (a) VS Code cannot file an identifier that I know is in the file and (b) the tooltip and the line under the mouse cursor don't match!   
![image](https://user-images.githubusercontent.com/18409827/133130047-f1bdb9b2-9daa-4fc6-8db3-6a8d28fa1259.png)

This is just a sample.  *See [the video](https://www.youtube.com/watch?v=Q2vG8PAK7Ek) for more!*

**Versions**

vscode: 1.60.0 deno: 1.13.2 extension: v3.8.1

Version: 1.60.0 (user setup)
Commit: e7d7e9a9348e6a8cc8c03f877d39cb72e5dfb1ff
Date: 2021-09-01T10:41:52.311Z
Electron: 13.1.8
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Windows_NT x64 10.0.19042

Starting Deno language server...
  version: 1.13.2 (release, x86_64-pc-windows-msvc)
  executable: C:\Users\Philip Smolen\.deno\bin\deno.EXE
Connected to "Visual Studio Code" 1.60.0
Enabling import suggestions for: https://deno.land
language server initialized
Server ready.

**Comment by TradeIdeasPhilip** [comment]
The problem went away.
I have no idea why.

I just ignored that part of my project and continued to fix and build other parts of the project.  I went back and I saw this:
![Screenshot (327)](https://user-images.githubusercontent.com/18409827/133700767-baf167a9-dba2-4e07-b139-5e7fc5b35ee7.png)
This is what I expected from the beginning.  It tells me that the file needs to be cached.

I clicked "quick fix" then took the option to cache the file.  Now I can use writeAll(), as expected.

I've seen a lot of weird stuff in VS code, but most of it goes away when you restart VS code.
